lib/network/modules/SelectionHandler.js

```diff
@@ -653,7 +653,14 @@ class SelectionHandler {
     }
 
     if (object !== undefined) {
-      hoverChanged = hoverChanged || this.emitHoverEvent(event, pointer, object);
+      const hoveredEdgesCound = Object.keys(this.hoverObj.edges).length;
```

Comment:
`hoveredEdgesCound` -> `hoveredEdgesCount`

Reply:
Good catch, thank :+1: I have fixed the typo